frontend/src/scenes/data-warehouse/settings/DataWarehouseSourceIcon.tsx

```diff
@@ -0,0 +1,93 @@
+import { Link } from 'lib/lemon-ui/Link'
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+import BlushingHog from 'public/hedgehog/blushing-hog.png'
+import IconAwsS3 from 'public/services/aws-s3.png'
+import Iconazure from 'public/services/azure.png'
+import IconBigQuery from 'public/services/bigquery.png'
+import IconChargebee from 'public/services/chargebee.png'
+import IconCloudflare from 'public/services/cloudflare.png'
+import IconGoogleCloudStorage from 'public/services/google-cloud-storage.png'
+import IconHubspot from 'public/services/hubspot.png'
+import IconMySQL from 'public/services/mysql.png'
+import IconPostgres from 'public/services/postgres.png'
+import IconSalesforce from 'public/services/salesforce.png'
+import IconSnowflake from 'public/services/snowflake.png'
+import IconMSSQL from 'public/services/sql-azure.png'
+import IconStripe from 'public/services/stripe.png'
+import IconVitally from 'public/services/vitally.png'
+import IconZendesk from 'public/services/zendesk.png'
+import { getDataWarehouseSourceUrl } from 'scenes/data-warehouse/settings/DataWarehouseManagedSourcesTable'
+
+/**
+ * In some cases we don't have the backend telling us what provider we have for blob storage, so we can have some
+ * heuristic to guess, then fallback to a shrugging hedgehog.
```

Comment:
Not one for now, but we should probably start storing this in the DB

Reply:
This is part of the reason I wanted to reference the actual table instead of piggybacking on the big /`query` response.